src/legacy/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -497,6 +497,7 @@ app.directive('dashboardApp', function ($injector) {
         showShareContextMenu({
           anchorElement,
           allowEmbed: true,
+          allowShortUrl: true,
```

Comment:
__nit:__ would you mind adding a comment in the code explaining why `allowShortUrl` is always `true`?

Reply:
Absolutely!